cou/cli.py

```diff
@@ -33,6 +33,9 @@
 COU_DIR_LOG = pathlib.Path(os.getenv("HOME", ""), ".local/share/cou/log")
 AVAILABLE_OPTIONS = "cas"
 
+# NOTE(gabrielcocenza) top-level module to reference the same logger object across modules.
+logger = logging.getLogger("cou")
```

Comment:
Why we could not use root logger to set "debug" log level?
Can we be more smart here and get module name or logger from module itself? Using `"cou"` sting it not look to good to me.






























`

Reply:
The problem that I faced here is that we are setting the log on a submodule of the project. If we use `__name__`here this would result in `cou.cli` and this is not the `top` module that will reference to the same logger object. If we were doing this on the `__init__.py` of `cou` than it wouldn't be a problem.

Using the root logger it's not advisable from the [docs](https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library).

> Logging to the root logger will make it difficult or impossible for the application developer to configure the logging verbosity or handlers of your library as they wish

Reply:
In this file we should use `logger = logging.getLogger(__name__)` to do any logging. So we need to update line `logging.info("Logs of this execution can be found at %s", file_name)` to `logger.info("Logs of this execution can be found at %s", file_name)`.
You are right it's not advisable to use root logger for any logging, but we need to use root logger for any configuration. Without usage of root logger, we will lost all logs from different libraries such are `libjuju`, `websockets`, ...
I know that from `libjuju` and `websockets` there are to many logs, so I'll recommend to use filter for console_handler to log only `cou` logs.